<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_dialog</class>
 <widget class="QDialog" name="main_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>main_dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="selCfgFilePB">
       <property name="text">
        <string>选择配置文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="selCfgFileLE"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="selSrcFolderPB">
       <property name="text">
        <string>选择源文件夹</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="selSrcFolderLE"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="selDstFolderPB">
       <property name="text">
        <string>选择目标文件夹</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="selDstFolderLE"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="startProcPB">
       <property name="text">
        <string>开始处理</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="clearMsgPB">
       <property name="text">
        <string>清空消息</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="logTE"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
